src/app/header/header.component.scss

```diff
@@ -1,11 +1,15 @@
+@import "src/assets/styles/variables";
 .header-container {
   padding: 0 24px;
   border: 1px solid rgba(0, 0, 0, 0.12);
   width: 97.2rem;
-  height: 72px;
+  height: 70px;
   background-color: floralwhite;
   &--btn {
     font-family: Roboto;
   }
 }
+.p,span {
```

Comment:
are you trying to target **p** tag ??  or is it a class ??

Reply:
both are tag....

Reply:
please give the proper class name instead of .p and span.

Reply:
resolved....................Thank you